.github/workflows/node.yml

```diff
@@ -118,6 +163,8 @@ jobs:
     lint-rust:
         timeout-minutes: 15
         runs-on: ubuntu-latest
+        needs: get-matrices
+        if: ${{ needs.get-matrices.outputs.should-run == 'true' }}
```

Comment:
same. why do we limit linter runs?

Reply:
We block run on different type of label, this way I'm avoiding any workflow to run on label which are not the label we wanted. Anyhow ill remove the label for now so ill remove this as well.